server/src/main/scala/io/delta/sharing/server/DeltaSharingService.scala

```diff
@@ -283,18 +301,22 @@ class DeltaSharingService(serverConfig: ServerConfig) {
       version = None,
       timestamp = None,
       startingVersion = None,
-      endingVersion = None
-    )
+      endingVersion = None,
+      responseFormat = capabilitiesMap.get(DELTA_SHARING_RESPONSE_FORMAT).getOrElse("parquet"))
```

Comment:
Should we consolidate default values in some central config?
conf.getDeltaSharingResponseFormat()